airflow/configuration.py

```diff
@@ -705,6 +705,28 @@ def _warn_deprecate(section, key, deprecated_section, deprecated_name):
                 stacklevel=3,
             )
 
+    def get_session_lifetime_config(self):
+        session_lifetime_days = self.getint('webserver', 'SESSION_LIFETIME_DAYS')
+        if session_lifetime_days or self.getint('webserver', 'FORCE_LOG_OUT_AFTER'):
```

Comment:
```suggestion
        session_lifetime_days = self.getint('webserver', 'session_lifetime_days', fallback=None)
        if session_lifetime_days or self.getint('webserver', 'force_logout_after', fallback=None):
```

Reply:
This will fix your test failures:

```
airflow.exceptions.AirflowConfigException: section/key [webserver/session_lifetime_days] not found in config
```